TUIFIManager/TUIMenu.py

```diff
@@ -1,6 +1,6 @@
 #from unicurses import  *
 import unicurses
-from TUItilities import BEGIN_MOUSE,END_MOUSE
```

Comment:
unused import, removed it

Reply:
Oopsie, I forgot those there, Thanks for the catch!